.github/container/nsys-jax-gather-src-files

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e -o pipefail
+
+# Handle commandline arguments
+proto_dir="$1"
+shift
+
+# Work in a temporary directory
+tmp_dir=$(mktemp -d --tmpdir protoc-install.XXXXXXXX)
+pushd "$tmp_dir" 1>&2
```

Comment:
Is there a way to globally redirect stdout to stderr?

Reply:
Note that it's not global; the Python command with the `EOL` delimiters prints to stdout.